testsuite/test-depmod.c

```diff
@@ -16,7 +16,8 @@
 
 #define EXEC_DEPMOD(...)                     \
 	test_spawn_prog(TOOLS_DIR "/depmod", \
-			(const char *[]){ TOOLS_DIR "/depmod", ##__VA_ARGS__, NULL })
+			(const char *[]){    \
+				TOOLS_DIR "/depmod" __VA_OPT__(, ) __VA_ARGS__, NULL })
```

Comment:
> C23 introduces __VA_OPT__, which has been supported by GCC/Clang since
> 2018. Let's use that instead of the GNU extension.

but what's the motivation to use it?
1) we are not going to change to stop using gnu extensions
2) `##__VA_ARGS__` is supported everywhere that matters
3) the standard alternative makes it harder to parse